Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -2949,6 +2957,8 @@
     allowMovement: true
     description: ghost-role-information-SyndiCat-description
     rules: ghost-role-information-SyndiCat-rules
+    mindRoles:
+    - MindRoleGhostRoleTeamAntagonist
```

Comment:
I thought they will be familiars too because I vaguely remembered so, but Headmin wanted them team antag
https://discord.com/channels/310555209753690112/1281461283157184532/1312355577388601404
"Familiars are non antags but are bound to follow the commands of the person who summoned them."